src/Controllers/PushController.js

```diff
@@ -71,6 +71,9 @@ export class PushController extends AdaptableController {
       let updateWhere = deepcopy(where);
 
       badgeUpdate = () => {
+        if (updateWhere.arn){
```

Comment:
@flovilmart this could be improved in the future, and as fair as I know, when sending bulk PN via Topic with SNS, it is not possible to increment the badge, otherwise you need to go through all installations and Endpoints again, which can lead to high memory usage. We still able to send bulk PN and increment the badge if we want by passing the Installation Query